### ISSUE TITLE
Rules - Full Game - Media Release

### ISSUE THREAD

**Issue by Stephen-Gates** [opendataboardgame/game#40]
Do media release cards to the current and subsequent turns until the media card is replaced with another?

"Government has introduced new legislation requiring that all weather data **currently on the board** be open" is confusing.

When playing, we assumed it continued to apply until replaced.

**Comment by ellenbroad** [comment]
I'm not sure what your first question is, but yes it continues to apply until replaced.  Have added a comment to the instructions here: https://docs.google.com/document/d/1isIfIu0ThbXjLFTxgKUhMoRCA6kpHMooLyJvSdNCEl4/edit

**Comment by Stephen-Gates** [comment]
@ellenbroad  Yeah sorry. Should of said "Do media release cards **apply**..." in the first question. sorry.
